src/Jobs/BasePackageJob.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Flagrow\Bazaar\Jobs;
+
+use Carbon\Carbon;
+use Flagrow\Bazaar\Composer\ComposerCommand;
+use Flagrow\Bazaar\Composer\ComposerEnvironment;
+use Flagrow\Bazaar\Composer\ComposerOutput;
+use Flagrow\Bazaar\Task;
+//use Illuminate\Bus\Queueable;
+//use Illuminate\Queue\SerializesModels;
+//use Illuminate\Queue\InteractsWithQueue;
+//use Illuminate\Contracts\Queue\ShouldQueue;
```

Comment:
I've actually started a PR against core to support queues, but there was too much confusion in the team to finish it up. I wouldn't add our own (yet), unless we need it asap.